CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contributors' Guide
+
+This guide will help familiarize contributors to the `containerd/containerd` repository.
+
+## Prerequisite
+
+First read the containerd project's [general guidelines around contribution](https://github.com/containerd/project/blob/main/CONTRIBUTING.md)
+which apply to all containerd projects.
+
+## Getting started
+
+See [`BUILDING.md`](https://github.com/containerd/containerd/blob/main/BUILDING.md) to setup a local development environment.
+
+If you are also a new user to containerd, you can first check out the [_Getting starting with containerd_](https://github.com/containerd/containerd/blob/main/docs/getting-started.md) guide.
+
+## Setting up your environment
```

Comment:
Putting myself in a new contributors shoes, I'd find it odd to read "follow this to setup a local development environment" and then a couple lines below (coming off of reading BUILDING.md lets say 😆) see "Setting up your environment".

Reply:
nod

there are a number of ways to set up the development environment and different requirements depending on what containerd based products you are working on, whether you want to use a devcontainer, what platforms you are working on, whether you are doing core api changes..